### ISSUE TITLE
In Dark mode, color of 'Change behavior' string is not A11Y compatible

### ISSUE THREAD

**Issue by rodrigosf672** [posit-dev/positron#10603]
<img width="660" height="176" alt="Image" src="https://github.com/user-attachments/assets/d39d9e58-0223-42a2-a2f7-5a41414a99cf" />

<img width="723" height="916" alt="Image" src="https://github.com/user-attachments/assets/221e624f-40e7-49a4-b8f0-0bc001399873" />

#### Positron and OS details:

```
Positron Version: 2025.12.0 build 72
Code - OSS Version: 1.105.0
Commit: 3af147dca07686d2cfa80faf0796c04661c56616
Date: 2025-11-14T07:19:22.022Z
Electron: 37.6.0
Chromium: 138.0.7204.251
Node.js: 22.19.0
V8: 13.8.258.32-electron.0
OS: Darwin arm64 25.0.0
```

## Describe the issue:
In Dark mode, color of 'Change behavior' string is not A11Y compatible

## Steps to reproduce the issue:

1. Switch to dark mode and create notebook code cell.
2. Add code that generates a lot of lines.
```python
for i in range(5000):
    print(f"line {i}")
```
3. See that the color of the 'Change behavior' string (#4a1b75) does NOT constrast well with the dark background (#1f1f1f), violating basic A11Y WCAG AA standards.

## Expected or desired behavior:
Color contrast should ensure basic accessibility and WCAG-compliance for users.

**Comment by rodrigosf672** [comment]
### Verified Fixed
    Positron Version(s) : 2025.12.0 build 152
    OS Version          : macOS Tahoe 26.1

### Test scenario(s) 

New dark mode for Notebooks is very neat, I really like it! Please find testing results below. 
- Normal text meets WCAG AA but fails WCAG AAA.
- Large text meets both WCAG AA and AAA.
- Graphical text meets WCAG AA.

<img width="732" height="912" alt="Image" src="https://github.com/user-attachments/assets/bed66051-9fae-4bbc-a645-0bc4a58b75c3" />
